crates/hc_sandbox/src/cmds.rs

```diff
@@ -292,6 +296,7 @@ impl Default for Create {
             directories: Vec::with_capacity(0),
             in_process_lair: false,
             no_dpki: false,
+            dpki_network_seed: None,
```

Comment:
As this is a conductor meant for testing and DPKI is enabled by default, I suggest to make a random string the default network seed.

Reply:
Will multiple sandbox conductors created in a single batch all share the same config? If so then I agree, if not, then perhaps this is fine. I would expect you would want those sandboxes to talk to each other and not be isolated.

Reply:
Currently, [looks like not](https://github.com/holochain/holochain/blob/40ac0cbc4a8407bbee9e403097d59ad1f09759ac/crates/hc_sandbox/src/sandbox.rs#L75). So we'd have to make sure batches all get the same random DPKI seed. Maybe worth doing, maybe not?

Reply:
@jost-s unfortunately not, if you drill in, you see that each iteration of the loop generates a new config from scratch without knowledge of the other configs

https://github.com/holochain/holochain/blob/develop/crates/hc_sandbox/src/generate.rs#L32

Reply:
Ah, unfortunate indeed. I'm concerned that a simple `hc s generate` will use no DPKI network seed and therefore share the "main" DPKI DNA. Not good for testing, but also not good for the production DPKI DNA.

Reply:
Okay, I'm fine with it.